### PR TITLE
Beets 447 single asset pages design

### PR DESCRIPTION
**Pull request by piratecody**
### Definition of Done:

* [X] Have you checked your [Jira](https://bee-cs458.atlassian.net/jira/software/projects/BEETS/boards/2/backlog) task, to make sure the user acceptance criteria are met?
* [X] Does your code have comments describing its function and reason?

### Source Control

* [X] Is all relevant code checked-in?
* [X] Is this branch ready to be reviewed by the Tech Lead?

### Formatting

* [X] Do your variable names make sense (to others)?
* [X] Have you removed unnecessary Console.logs? 
* [X] Is your code style consistent with other code in the file or the [Style Guide](https://google.github.io/styleguide/jsguide.html)

### CODE REVIEW

**Review by levertyk — CHANGES_REQUESTED** [review]
When I log in as an operator, and select an asset from the list, the page crashes.
Screenshot 2023-04-11 at 11 09 05 AM
Screenshot 2023-04-11 at 11 09 23 AM

**Comment by levertyk** [comment]
> When I log in as an operator, and select an asset from the list, the page crashes. Screenshot 2023-04-11 at 11 09 05 AM Screenshot 2023-04-11 at 11 09 23 AM

The same thing happens as an owner.

**Comment by piratecody** [comment]
> > When I log in as an operator, and select an asset from the list, the page crashes. Screenshot 2023-04-11 at 11 09 05 AM Screenshot 2023-04-11 at 11 09 23 AM
> 
> The same thing happens as an owner.

@levertyk I can't seem to replicate the issue. Could you let me know what asset you were trying to view?

File: client/src/components/AssetTable/AssetRow/AssetRow.js [hunk 1]
```diff
@@ -134,14 +136,14 @@ function AssetRow(props) {
           </Button>
         </td>
       </AccessControl>
-      <td><Link to={`/asset/${asset.asset_tag}`} state={{ asset }}>{asset.asset_tag}</Link></td>
+      <td><Link to={`/asset/${asset.asset_tag}`} state={{ asset, categoryLabel }}>{asset.asset_tag}</Link></td>
```
[the remark this answers is not in the document]
Comment (piratecody) [reply client/src/components/AssetTable/AssetRow/AssetRow.js:139]: I can't replicate this either :/

**Comment by piratecody** [comment]
@levertyk I may have found the issue & patched it. please try again

**Comment by levertyk** [comment]
> @levertyk I may have found the issue & patched it. please try again

@piratecody 
I'm still getting issues, but now with more error messages. It happens on all assets, or at least the 8 assets I checked.

Screenshot 2023-04-12 at 10 28 21 AM
Screenshot 2023-04-12 at 10 28 12 AM

**Comment by piratecody** [comment]
> > @levertyk I may have found the issue & patched it. please try again
> 
> @piratecody I'm still getting issues, but now with more error messages. It happens on all assets, or at least the 8 assets I checked.
> 
> Screenshot 2023-04-12 at 10 28 21 AM
> 
> Screenshot 2023-04-12 at 10 28 12 AM

I do see one issue - currently only owners can see checkout records. Do we want operators to be able to see the checkout history?

**Comment by piratecody** [comment]
> @piratecody Yeah, they should. Was that hard coded into the API call?

Yeah, the api endpoint to get all checkout records is restricted to owners right now. I can change that and some of the errors should clear up.

**Comment by piratecody** [comment]
> > > @piratecody Yeah, they should. Was that hard coded into the API call?
> > 
> > 
> > Yeah, the api endpoint to get all checkout records is restricted to owners right now. I can change that and some of the errors should clear up.
> 
> Yeah, let's do that. I think that was just a logical error then.

Alright changed that. Also, the reason the cart column is so big for operators is that the edit asset column doesn't appear for them, so the columns expand to fill the space (I think)

**Review by levertyk — APPROVED** [review]

**Pull request merged**